src/registrar/fixtures.py

```diff
@@ -56,9 +59,32 @@ class UserFixture:
         },
     ]
 
+    STAFF = [
+        {
+            "username": "319c490d-453b-43d9-bc4d-7d6cd8ff6844",
+            "first_name": "Rachid-Analyst",
+            "last_name": "Mrad-Analyst",
+        },
```

Comment:
```suggestion
        },
        {
            "username": "b6a15987-5c88-4e26-8de2-ca71a0bdb2cd",
            "first_name": "Alysia-Analyst",
            "last_name": "Alysia-Analyst",
        },
```
If you dont mind adding this to your PR, otherwise I have the change saved and can make a PR to add myself

Reply:
Happy to add to my PR :)